### PR TITLE
fixes #30

### LINKED ISSUE #30: Fails on windows: pypiwin32 outdated fork is used.
For some reason the windows related dependency `pypiwin32`  - an outdated fork of `pywin32` - fails to install in Plone's `buildout.coredev`.

```
Installing instance.
While:
  Installing instance.
  Getting distribution for 'pypiwin32'.
Error: Picked: pypiwin32 = 223
Write to Town Square
```

It seems reason of the fork and its usage was a stalled project. It seems the pywin32 project is now maintained again. There are newer releases and commits. 

I propose to use the main project again. We will try if usage of the main project release fixes our problem and in case it works we will prepare a PR.

cc @nilshofer

### PR DESCRIPTION
**Pull request by jensens**
Use ``pywin32`` again, not any longer the meanwhile outdated fork named ``pypiwin32``.
 Add some information for installation with buildout.

### CODE REVIEW

File: src/zope/sendmail/README.rst [hunk 1]
```diff
@@ -7,6 +7,14 @@ integrates with the transaction mechanism and queues your emails to be sent on
 successful commits only.
 
 
+Install on Windows/ Buildout
+============================
+
+When using Windows, `pywin32 <https://github.com/mhammond/pywin32>`_ is required.
+``pywin32`` can only be installed using Pip, it **does not work with ``zc.buildout``**.
```
Comment (jugmac00) [inline src/zope/sendmail/README.rst:14]:
```
Pip -> ``pip``
```

File: src/zope/sendmail/README.rst [hunk 2]
```diff
@@ -7,6 +7,14 @@ integrates with the transaction mechanism and queues your emails to be sent on
...
+When using buildout, prior to execution of buildout, use to install ``pywin32``.
+
```
Comment (jugmac00) [inline src/zope/sendmail/README.rst:16]:
```
use to install ``pywin32``.
=>
use ``pip`` to install ``pywin32``.
```

File: src/zope/sendmail/README.rst [hunk 3]
```diff
@@ -7,6 +7,14 @@ integrates with the transaction mechanism and queues your emails to be sent on
 successful commits only.
 
 
+Install on Windows/ Buildout
+============================
+
+When using Windows, `pywin32 <https://github.com/mhammond/pywin32>`_ is required.
+``pywin32`` can only be installed using Pip, it **does not work with ``zc.buildout``**.
+When using buildout, prior to execution of buildout, use to install ``pywin32``.
```
Comment (jugmac00) [inline src/zope/sendmail/README.rst:15]: prior to execution of buildout
=>
prior to the execution of buildout

File: src/zope/sendmail/README.rst [hunk 4]
```diff
@@ -7,6 +7,14 @@ integrates with the transaction mechanism and queues your emails to be sent on
 successful commits only.
 
 
+Install on Windows/ Buildout
```
Comment (jugmac00) [inline src/zope/sendmail/README.rst:10]: Install on Windows/ Buildout
=>
Installation on Windows/ Buildout

File: src/zope/sendmail/README.rst (hunk 1 above)
Comment (jensens) [reply src/zope/sendmail/README.rst]: thx, changed

File: src/zope/sendmail/README.rst (hunk 2 above)
Comment (jensens) [reply src/zope/sendmail/README.rst:16]: thx, changed

File: src/zope/sendmail/README.rst (hunk 3 above)
Comment (jensens) [reply src/zope/sendmail/README.rst]: thx, changed

File: src/zope/sendmail/README.rst (hunk 4 above)
Comment (jensens) [reply src/zope/sendmail/README.rst]: thx, changed

**Comment by jensens** [comment]
@jugmac00 Thanks! So we (@nilshofer and I) tested it on Windows 10 already and it seems to work.

**Pull request merged**